advanced-achievements-plugin/src/main/java/com/hm/achievement/command/completer/CommandTabCompleter.java

```diff
@@ -56,49 +65,53 @@ public void extractConfigurationParameters() {
 
 		enabledCategoriesWithSubcategories.clear();
 		for (MultipleAchievements category : MultipleAchievements.values()) {
-			for (String subcategory : mainConfig.getShallowKeys(category.toString())) {
-				enabledCategoriesWithSubcategories.add(category + "." + StringUtils.deleteWhitespace(subcategory));
+			if (!disabledCategories.contains(category.toString())) {
+				for (String subcategory : mainConfig.getShallowKeys(category.toString())) {
+					enabledCategoriesWithSubcategories.add(category + "." + StringUtils.deleteWhitespace(subcategory));
+				}
 			}
 		}
 		for (NormalAchievements category : NormalAchievements.values()) {
-			enabledCategoriesWithSubcategories.add(category.toString());
+			if (!disabledCategories.contains(category.toString())) {
+				enabledCategoriesWithSubcategories.add(category.toString());
+			}
 		}
-		enabledCategoriesWithSubcategories.add("Commands");
```

Comment:
_Commands_ category was removed from `enabledCategoriesWithSubcategories`, is this intentional? :)

Reply:
Yes, this is intentional, I should have listed it in the improvements above. _/aach_ reset and _/aach add_ can simply not be used with the _Commands_ category. 😉